src/Contracts/Providers/Composer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * This file is part of Zero Framework.
+ *
+ * (c) Nuno Maduro <enunomaduro@gmail.com>
+ *
+ *  For the full copyright and license information, please view the LICENSE
+ *  file that was distributed with this source code.
+ */
+
+namespace NunoMaduro\ZeroFramework\Contracts\Providers;
+
+/**
+ * The is the Zero Framework composer contract.
```

Comment:
Did you mean "*This* is the Zero Framework composer contract."? 😉

Reply:
@svenluijten Yeah. Nice catch 👍